pages/complete.tsx

```diff
@@ -0,0 +1,346 @@
+import NavBar from '../components/Critical/NavBar'
+import styled from 'styled-components'
+import SideProgressBar from '../components/Critical/SideProgressBar'
+import { useRouter } from 'next/router'
+import {
+  NextEndButton,
+  BottomButtonBar,
+  BackButton,
+  ButtonContainer,
+} from '../components/FormStyles/ExtraStyles'
+import { BlockQuote } from '../components/FormStyles/QuestionText'
+import { jsPDF } from 'jspdf'
+import { QuestionType } from '../models/question'
+import { FormResult } from '../utils/FormContext'
+import { studentSpecialCircumstances } from '../constants/additionalConstants'
+import { AdditionalInfoDb } from './api/form/additionalinfo/save'
+import { ConcernDB } from './api/form/concern/save'
+import { ContactInfoDb } from './api/form/contactinfo/save'
+import { DistrictDB } from './api/form/district/save'
+import { GroupDB } from './api/form/group/save'
+import { FormTemplate } from '../components/Critical/FormTemplate'
+
+const Main = styled.div`
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+  align-items: stretch;
+`
+
+const LowerRight = styled.div`
+  display: flex;
+  flex-direction: column;
+`
+const LowerStuff = styled.div`
+  display: flex;
+  flex-direction: row;
+  height: 100vh;
+`
+const InfoSection = styled.div`
+  display: flex;
+  flex-direction: column;
+  height: 80vh;
+`
+
+const SideBarStyled = styled(SideProgressBar)`
+  display: flex
+  flex-grow: 1
+`
+
+const StyledBottomButtonBar = styled(BottomButtonBar)`
+  display: flex
+  align-self: flex-end
+
+`
+
+// determines whether the given doc needs a new page added to it
+// y denotes where the cursor is placed on the doc in terms of height / the 'y' axis
+// since jsPDF is a little bitch (but better than others)
```

Comment:
ok i know i left these comments but can we get rid of the mean ones pls (or keep them for us to look back on and laugh lmao)

Reply:
highkey did not notice these were there 🤭